app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -9,7 +9,7 @@ class Api::V2::ActivationKeysController < Api::V2::ApiController
     before_action :find_activation_key, :only => [:show, :update, :destroy, :available_releases, :copy, :product_content,
                                                   :available_host_collections, :add_host_collections, :remove_host_collections,
                                                   :content_override, :add_subscriptions, :remove_subscriptions,
-                                                  :subscriptions]
+                                                  :subscriptions, :content_overrides]
```

Comment:
Should it be content_override or content_overrides?

Reply:
@parthaa  - :content_override is already in there, but is being deprecated for content_overrides

Reply:
@cfouant its "action" name here not the param name. You are correct in that content override parameter is deprecated but content_override is the name of the action,